### DISCUSSION TITLE
mmp architecture

### DISCUSSION THREAD

**Discussion by Nightcor157** [OpenRadioss/OpenRadioss#1701] [Q&A] [answered]
Hello, I recently started learning OpenRadios and I had a question, is the mmp architecture supported? when building, I only see a mention of smp, but suddenly I'm missing something

**Comment by PaulAltair** [comment, marked as answer]
Yes, see:  https://github.com/OpenRadioss/OpenRadioss/blob/main/HOWTO.md

For how to build for/with open mpi under Linux. 

(You run the build script separately with ompi options to compile for ompi, similar for extended single precision version)

There are also pre-compiled binaries available for intel MPI version under windows
